src/codegen/c.zig

```diff
@@ -464,7 +464,7 @@ pub const DeclGen = struct {
                     .variable => ptr_val.castTag(.variable).?.data.owner_decl,
                     else => unreachable,
                 };
-                try dg.renderDeclValue(writer, decl.ty, ptr_val, decl);
+                try dg.renderDeclName(writer, decl);
                 return decl.ty;
             },
             .field_ptr => {
```

Comment:
I think I'm the cause of this bug, so thanks for fixing this 🙂 

It seems like `.field_ptr` would need to a dereference too, right?

Reply:
You're right, thanks for pointing that out. The PR was merged before I noticed your comment, so I'm just going to open a new PR with the fix :+1:

Reply:
> The PR was merged before I noticed your comment

Sorry about that, I misread topolarity's comment.